app/Filament/Resources/EmulatorResource/RelationManagers/EmulatorReleasesRelationManager.php

```diff
@@ -57,10 +64,11 @@ public function table(Table $table): Table
 
             ])
             ->headerActions([
-
+                Tables\Actions\CreateAction::make(),
```

Comment:
Should this create an audit log entry?

Reply:
Updated in latest to begin recording these logs, however there isn't a dedicated page for them at the moment (see other comment below).